src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -216,6 +216,11 @@ export default class DefaultAudioVideoController
     return this._eventController;
   }
 
+  /**
+   * This API will be deprecated in favor of `ClientMetricReport.getRTCStatsReport()`.
+   * Please subscribe to `metricsDidReceive(clientMetricReport: ClientMetricReport)` event,
+   * and get the raw `RTCStatsReport` via `clientMetricReport.getRTCStatsReport()`.
+   */
   getRTCPeerConnectionStats(selector?: MediaStreamTrack): Promise<RTCStatsReport> {
     if (!this.rtcPeerConnection) {
```

Comment:
can we do a logger.warn here and print out a message?

Reply:
Good idea, will do

Reply:
Added